src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -3218,12 +3247,78 @@ public ConstructorEdit(INamedTypeSymbol oldType)
             }
         }
 
+        private static bool TryAddParameterInsertOrDeleteEdits(ArrayBuilder<SemanticEditInfo> semanticEdits, ISymbol containingSymbol, SemanticModel? otherModel, EditAndContinueCapabilitiesGrantor capabilities, Func<SyntaxNode, SyntaxNode?>? syntaxMap, EditScript<SyntaxNode> editScript, CancellationToken cancellationToken, out bool notSupportedByRuntime)
+        {
+            notSupportedByRuntime = false;
+
+            // Since we're inserting (or deleting) a parameter node, oldSymbol (or newSymbol) would have been null,
+            // and a symbolkey won't map to the other compilation because the parameters are different, so we have to go back to the edit map
+            // to find the declaration that contains the parameter, and its partner
+
+            // Some sanity checks
+            if (otherModel is null ||
+                containingSymbol.DeclaringSyntaxReferences.Length != 1)
```

Comment:
>containingSymbol.DeclaringSyntaxReferences.Length != 1 [](http://example.com/codeflow?start=16&length=54)

Does this exclude partial methods? Doc please.

Reply:
This part wasn't a problem with partial methods, so maybe I missed something, but I added a test, and another check to make the test pass :)